hub/core/tensor.py

```diff
@@ -6,20 +6,31 @@
 from hub.core.sample import Sample  # type: ignore
 from hub.core.chunk_engine import ChunkEngine, SampleValue
 from hub.api.info import load_info
-from hub.util.keys import get_tensor_meta_key, tensor_exists, get_tensor_info_key
+from hub.core.meta.hashlist import load_hashlist
+from hub.core.meta.hashlist import Hashlist
+from hub.util.keys import (
+    get_tensor_meta_key,
+    get_hashlist_key,
+    hashlist_exists,
+    tensor_exists,
+    get_tensor_info_key,
+)
 from hub.util.shape import ShapeInterval
 from hub.util.exceptions import (
     TensorDoesNotExistError,
     InvalidKeyTypeError,
     TensorAlreadyExistsError,
+    HashlistAlreadyExistsError,
```

Comment:
this isn't called anywhere I think, should be called if you try to treat 2 tensors as hashlists